### DISCUSSION TITLE
fr.json

### DISCUSSION THREAD

**Discussion by Lanneval** [AnimaBeyondDevelop/AnimaBeyondFoundry#59] [Ideas]
Hello,

I have send you the fr.json file for Anima Beyonf Fantasy.

thank you for this system.

Sorry for my english, i'm a french guy.

Arnaud

**Comment by DavidDPG** [comment]
Hey @Lanneval, i've created an issue to add french: #60.

**Comment by DavidDPG** [comment]
Created #65, @Lanneval once it's merged and published we'd appreciate you checking it out 😃
